viz/cmd/check.go

```diff
@@ -71,23 +71,27 @@ func configureAndRunChecks(wout io.Writer, werr io.Writer, options *checkOptions
 		return fmt.Errorf("validation error when executing check command: %w", err)
 	}
 
-	hc := vizHealthCheck.NewHealthChecker([]healthcheck.CategoryID{}, &healthcheck.Options{
-		ControlPlaneNamespace: controlPlaneNamespace,
-		KubeConfig:            kubeconfigPath,
-		KubeContext:           kubeContext,
-		Impersonate:           impersonate,
-		ImpersonateGroup:      impersonateGroup,
-		APIAddr:               apiAddr,
-		RetryDeadline:         time.Now().Add(options.wait),
-		DataPlaneNamespace:    options.namespace,
+	hc := vizHealthCheck.NewHealthChecker([]healthcheck.CategoryID{}, &vizHealthCheck.VizOptions{
+		Options: &healthcheck.Options{
+			ControlPlaneNamespace: controlPlaneNamespace,
+			KubeConfig:            kubeconfigPath,
+			KubeContext:           kubeContext,
+			Impersonate:           impersonate,
+			ImpersonateGroup:      impersonateGroup,
+			APIAddr:               apiAddr,
+			RetryDeadline:         time.Now().Add(options.wait),
+			DataPlaneNamespace:    options.namespace,
+		},
+		VizNamespaceOverride: vizNamespace,
 	})
 	err = hc.InitializeKubeAPIClient()
 	if err != nil {
 		fmt.Fprintf(werr, "Error initializing k8s API client: %s\n", err)
 		os.Exit(1)
 	}
 
-	hc.AppendCategories(hc.VizCategory())
+	fullCheck := true
+	hc.AppendCategories(hc.VizCategory(fullCheck))
```

Comment:
When reading this I assumed that it was done this way because go doesn't allow passing function arguments by name (e.g. `hc.VizCategory(fullcheck = true)`) and doing it this way hints at the semantics of the parameter at the call site.  We don't use this style elsewhere in this codebase, but I'm also not opposed to it.

Reply:
I know it didn't look quite well, I thought it looked a bit like "documenting" the parameter, but it's perfectly fine doing it directly.